### DISCUSSION TITLE
Running on non-Windows platforms?

### DISCUSSION THREAD

**Discussion by mitcharoni** [coldino/Larkator#56] [Q&A]
Do you think it's possible to run Larkator on non-Windows platforms that have .NET support like Linux or macOS?

**Comment by coldino** [comment]
Currently the .Net support for those platforms does not include any of the standard .Net UI systems (Larkator use WPF), meaning your only option would be trying to run it in WINE. I'd love to hear back if you try this.

However, Microsoft are releasing a cross-platform UI system later this year that will cover Mac OS as standard (and I'm going to assume Linux later via community contributions). I do intend to rework Larkator onto this platform as it will also open up the option of running on a phone/tablet.
